iothub/device/src/ModuleClient.cs

```diff
@@ -669,6 +669,15 @@ public Task<DeviceStreamRequest> WaitForDeviceStreamRequestAsync()
         public Task<DeviceStreamRequest> WaitForDeviceStreamRequestAsync(CancellationToken cancellationToken)
             => this.internalClient.WaitForDeviceStreamRequestAsync(cancellationToken);
 
+        /// <summary>
+        /// Accepts a Device Stream request.
+        /// </summary>
+        /// <param name="request">The Device Stream request received through </param>
+        /// <returns>A awaitable async task</returns>
+        public Task AcceptDeviceStreamRequestAsync(DeviceStreamRequest request)
```

Comment:
>DeviceStreamRequest [](start = 51, length = 19)

If these requests can go to modules, should they still be called DeviceStreamRequest?

Reply:
Otherwise this all looks good to me.

---
In reply to: [251634557](https://github.com/Azure/azure-iot-sdk-csharp/pull/772#discussion_r251634557) [](ancestors = 251634557)